src/app/componments/header/LocaleButton.tsx

```diff
@@ -0,0 +1,14 @@
+import { LocaleButtonProps } from './LocaleButton.types';
+import * as Styled from './LocaleButton.styles';
+import { AppLocale } from '../../../context/locale/appLocale.enum';
+
+export const LocaleButton = ({ localeButtonValue, hasBorder }: LocaleButtonProps) => {
+  return (
```

Comment:
Na przyszłość - jeśli masz funkcję, która zwraca tylko komponent bez dodatkowej logiki, to można to zapisać bardziej skrótowo:
```
export const LocaleButton = ({ localeButtonValue, hasBorder }: LocaleButtonProps) => (
  <>
    ...
  </>
);